### ISSUE TITLE
About rotation normalize

### ISSUE THREAD

**Issue by cdfan0627** [hbb1/2d-gaussian-splatting#134]
Hi 非常感謝你release code，可以請問一下為什麼下面這行code的activation function 不用設dim=1嗎，因為理論上來說四元數要normalize應該是要normalize在dim=1，而不是預設的dim=0嗎?
https://github.com/hbb1/2d-gaussian-splatting/blob/main/scene/gaussian_model.py#L101

**Comment by hbb1** [comment]
Hi, [torch.nn.functional.normalize](https://pytorch.org/docs/stable/generated/torch.nn.functional.normalize.html) 预设的 dim 是 1 喔！

**Comment by cdfan0627** [comment]
不好意思我搞錯了，謝謝!
